packages/istio/charts/istio-1.6.3/templates/post-install-job.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Release.Name }}-pod
+  namespace: {{ .Release.Namespace }}
+spec:
+  backoffLimit: 1
+  template:
+    spec:
+      containers:
+        - name: {{ .Release.Name }}-container
+          image: brendarearden/istio-test:1.6.0
+          env:
+          - name: RELEASE_NAME
+            value: {{ .Release.Name }}
+          command: ["/bin/sh","-c"]
+          args: ["./usr/local/app/scripts/run.sh"]
+          volumeMounts:
+          - name: config-volume
+            mountPath: /app/config.yaml
+            subPath: istio-base.yaml
+      volumes:
+        - name: config-volume
+          configMap:
+            name: {{ .Release.Name }}-configmap
```

Comment:
Same here, don't need to namespace with release name, its all contained within the job. Also might make it more descriptive like `istio-operator-cr` or `profile` or something.